gpii/node_modules/gsettingsBridge/tests/gsettingsTests.js

```diff
@@ -193,6 +193,8 @@ jqUnit.test("Getting single keys via gsettings handler", function () {
         gpii.gsettings.getSingleKey("net.gpii.testing.gsettings.single-get", "boolean-setting"));
     jqUnit.assertEquals("Checking 'string' key", "abcdefg",
         gpii.gsettings.getSingleKey("net.gpii.testing.gsettings.single-get", "string-setting"));
+    jqUnit.assertEquals("Checking non-existent' key", undefined,
```

Comment:
Good catch @javihernandez.  What's actually missing here is the type of the key.  It should be 'string' in this case. Fixed.